app/controllers/reactions_controller.rb

```diff
@@ -57,7 +57,7 @@ def create
   end
 
   def cached_user_public_comment_reactions(user, comment_ids)
-    cache = Rails.cache.fetch("cached-user-#{user.id}-reaction-ids-#{user.public_reactions_count}",
+    cache = Rails.cache.fetch("#{user.cache_key}/public-comment-reactions-count-#{user.public_reactions_count}",
```

Comment:
I'd suggest `public-comment-reactions` over `public-comment-reactions-count` for the main part of the key, since what we're actually caching here is the user's reactions

Reply:
@filleduchaos,

Nice suggestion.